integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -279,4 +280,16 @@ public boolean supportsSimilarity(VectorSimilarity vectorSimilarity) {
 				return true;
 		}
 	}
+
+	@Override
+	public boolean supportQueryStringRewriteMethod(RewriteMethod rewriteMethod) {
+		return isActualVersion(
+				es -> true,
+				// OpenSearch only has 6 options for the rewrites available:
+				//   constant_score, scoring_boolean, constant_score_boolean,
+				//   top_terms_N, top_terms_boost_N, and top_terms_blended_freqs_N
+				os -> !RewriteMethod.CONSTANT_SCORE_BLENDED.equals( rewriteMethod ),
+				aoss -> !RewriteMethod.CONSTANT_SCORE_BLENDED.equals( rewriteMethod )
```

Comment:
Turns out that this rewrite method is not available on the OpenSearch side..
Since this (rewrite methods) is probably an advanced option anyway, would it be enough to simply leave a note in the Javadocs for users to see what options are allowed by their backends, or should we do something similar to what was done for vector search, where each distribution would have own list of allowed methods and throw an early exception if something unsupported is supplied? 
Right now if constant score blended is supplied to OpenSearch it results in:
```
Response: 400 'Bad Request' from 'http://localhost:32769' with body 
{
  "error": {
    "root_cause": [
      {
        "type": "query_shard_exception",
        "reason": "failed to create query: Failed to parse rewrite_method [constant_score_blended]",
        "index": "indexname-000001",
        "index_uuid": "sm4DikTfSyyMyxLercR4fQ"
      }
    ],
    "type": "search_phase_execution_exception",
    "reason": "all shards failed",
    "phase": "query",
    "grouped": true,
    "failed_shards": [
      {
        "shard": 0,
        "index": "indexname-000001",
        "node": "sjCTMoEDR_CxWrvQkOX4Ng",
        "reason": {
          "type": "query_shard_exception",
          "reason": "failed to create query: Failed to parse rewrite_method [constant_score_blended]",
          "index": "indexname-000001",
          "index_uuid": "sm4DikTfSyyMyxLercR4fQ",
          "caused_by": {
            "type": "illegal_argument_exception",
            "reason": "Failed to parse rewrite_method [constant_score_blended]"
          }
        }
      }
    ]
  },
  "status": 400
}
```
at query execution time ....

Or just drop the option?